iml-command-utils/src/lib.rs

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+pub mod display_utils;
+
+#[cfg(feature = "cli")]
```

Comment:
Things that are used for `cli` should be made optional in the `Cargo.toml` and added to that feature.

Reply:
Done